server-base/src/main/java/org/apache/kylin/rest/job/StorageCleanupJob.java

```diff
@@ -217,25 +217,24 @@ private void cleanUnusedHdfsFiles() throws IOException {
     
     protected void collectUnusedHdfsFiles(UnusedHdfsFileCollector collector) throws IOException {
         if (StringUtils.isNotEmpty(config.getHBaseClusterFs())) {
-            cleanUnusedHdfsFiles(hbaseFs, collector);
+            cleanUnusedHdfsFiles(hbaseFs, collector, true);
         }
-        cleanUnusedHdfsFiles(defaultFs, collector);
+        cleanUnusedHdfsFiles(defaultFs, collector, false);
```

Comment:
Why should run cleanUnusedHdfsFiles twice if config.getHBaseClusterFs() is not empty?